backend/apis/parse-docket-pdf.utils.js

```diff
@@ -85,7 +85,17 @@ function parseCharges(lines, sections) {
 
   walkSection(lines, sections, "CHARGES", addDispositionToCharges);
 
-  return charges;
+  const chargesWithSeverity = charges.map(charge => {
+    const { description, rest } = charge;
```

Comment:
```suggestion
    const { description, ...rest } = charge;
```

Using rest args isn't even necessary here, though. See my comment on line 94